### PR TITLE
Adding server generation

### PR DESCRIPTION
**Pull request by depohmel**
modifing swagger to support /store
adding ImageMeta and ImageData models
adding serevr generation in build.sh

### CODE REVIEW

File: go/src/store/local/local_test.go [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+	c.Assert(err, IsNil)
+	c.Assert(img.Meta.Location, Equals, s.testImage.Meta.Location)
+}
+func (s *LocalStoreSuite) TestBLocalRead(c *C) {
+	img, err := s.storer.Read(context.TODO(), s.testImage.Meta)
+	c.Assert(err, IsNil)
+	c.Assert(img, Equals, s.testImage.Base64)
+}
+
+func (s *LocalStoreSuite) TestCLocalDelete(c *C) {
```
Comment (ashmrtn) [inline go/src/store/local/local_test.go:66]: is this test guaranteed to either try to delete a file that doesn't exist or a file that exists or could it be either case?

File: go/src/metaserver/configure_meta.go [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+	//
+	// Example:
+	// api.Logger = log.Printf
+
+	api.JSONConsumer = runtime.JSONConsumer()
+
+	api.JSONProducer = runtime.JSONProducer()
+
+	api.AddImageHandler = operations.AddImageHandlerFunc(func(params operations.AddImageParams) middleware.Responder {
+		return middleware.NotImplemented("operation .AddImage has not yet been implemented")
```
Comment (ashmrtn) [inline go/src/metaserver/configure_meta.go:39]: are these handlers supposed to be empty?

File: swagger.yaml [hunk 3]
```diff
@@ -22,7 +22,18 @@ basePath: /v0
 # items
 definitions:
   Image:
-    description: "Image is an objects represents picture in gallery"
+    description: "Image is an objects represents image in galerry"
```
Comment (ashmrtn) [inline swagger.yaml]:
```suggestion
    description: "Image is an object representing an image in gallery"
```

File: swagger.yaml [hunk 4]
```diff
@@ -22,7 +22,18 @@ basePath: /v0
...
+      meta:
+        $ref: '#/definitions/ImageMeta'
+      base64:
+        $ref: '#/definitions/ImageData'
+  ImageList:
+    type: array
+    items:
+      $ref: '#/definitions/ImageMeta'
+  ImageMeta:
+    description: "ImageMeta is an objects represents image meta data"
```
Comment (ashmrtn) [inline swagger.yaml]:
```suggestion
    description: "ImageMeta is an object representing image meta data"
```

File: go/src/store/local/local_test.go (hunk 1 above)
Comment (depohmel) [reply go/src/store/local/local_test.go:66]: yeap. Bug :)

File: go/src/metaserver/configure_meta.go (hunk 2 above)
Comment (depohmel) [reply go/src/metaserver/configure_meta.go:39]: i'm implementing it now. :)

File: go/src/store/local/local_test.go (hunk 1 above)
Comment (depohmel) [reply go/src/store/local/local_test.go:66]: Actually I named tests the way that file is always will be there

Comment (ashmrtn) [reply go/src/store/local/local_test.go:66]: alright, I'm not super familiar with how `gotest` works and whether it has a stable order to running tests, so I thought I'd double check :)

Comment (depohmel) [reply go/src/store/local/local_test.go:66]: it does not but naming tests liek TeastAsome, TestBsome, TestCsome will give you order

**Review by ashmrtn — APPROVED** [review]

**Pull request merged**